recipe/build.sh

```diff
@@ -1,4 +1,5 @@
-set -x
+#!/bin/bash
+set -ex
```

Comment:
If you're at it, let's make it as strict as we normally are.
```suggestion
set -exuo pipefail
```